README.md

```diff
@@ -1,19 +1,26 @@
 # Overview
 The qseed application downloads quantum entropy from Qrypt's EaaS and injects it into an HSM as seed random. The download and injection are performed periodically.
 
-See https://docs.qrypt.com/eaas/pkcs11/ for more information.
+See [Seed PKCS#11 HSMs](https://docs.qrypt.com/eaas/pkcs11/) for more information.
   
+## Table of Contents
+1. [Build](#build)
+2. [Quickstart using Thales HSMs](#quickstart-using-thales-hsms)
+3. [Test](#test)
+    1. [Build and Run GTests](#build-and-run-gtests)
+    2. [Test using SoftHSM](#test-using-softhsm)
+
 # Build
-This section covers how to build and install the qseed application. Note that you will need an installed cryptoki library on your system.
+This section covers how to build and install the qseed application. 
 
-1.  Create and navigate to a build directory.
-    ```bash
-    mkdir build && cd build
-    ```
+1.  Make sure you have the following library dependencies installed on your system. See the provided dockerfiles for installation details.
```

Comment:
Andrew - should we include installation instructions in the README for people who aren't using the Dockerfiles?